### PR TITLE
feat(workflow-dev): Add nsq to the workflow chart

### PR DESCRIPTION
**Pull request by jchauncey**
NSQ is a distributed messaging system build by bit.ly
It will provide us with the foundation to perform asynchronous communication across the platform. Especially in areas like Logging and Monitoring.

### CODE REVIEW

File: workflow-dev/tpl/deis-nsqd-rc.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      labels:
+        app: deis-nsqd
+    spec:
+      serviceAccount: deis-nsqd
+      containers:
+        - name: deis-nsqd
+          image: quay.io/{{.nsqd.org}}/nsqd:{{env "NSQ_GIT_TAG" | default .nsqd.dockerTag}}
+          imagePullPolicy: {{.nsqd.pullPolicy}}
+          command:
+          - /opt/nsq/bin/start-nsqd
```
Comment (arschles) [inline workflow-dev/tpl/deis-nsqd-rc.yaml:24]: feels like this would be more flexible if the start script were on the `PATH`

**Comment by arschles** [comment]
I think my comment at https://github.com/deis/charts/pull/287#discussion_r66883856 is not blocking for this PR. Code LGTM

**Comment by bacongobbler** [comment]
so the upgrade path moving forward will be ok with this change? all existing data will still exist, it's just the communication between logger and fluentd right? :)

**Comment by jchauncey** [comment]
Re upgrade path: that data is already lost when we upgrade so nothing to
consider on that for now. However we should look into making it more robust
in the future.

File: workflow-dev/tpl/deis-nsqd-rc.yaml (hunk 1 above)
Comment (jchauncey) [inline workflow-dev/tpl/deis-nsqd-rc.yaml:24]: so i toyed with putting it in `/bin` but felt this was more in line with how we do our other images.

Comment (krancour) [inline workflow-dev/tpl/deis-nsqd-rc.yaml:24]: I agree that this is consistent with how many of the other images look.

File: workflow-dev/tpl/deis-nsqd-rc.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#helm:generate helm tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifests/deis-nsqd-rc.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-nsqd
+  namespace: deis
+  labels:
+    heritage: deis
+    heritage: deis
```
Comment (krancour) [inline workflow-dev/tpl/deis-nsqd-rc.yaml]: Double `heritage` label?

**Pull request merged**